libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/hwdef.dat

```diff
@@ -233,4 +233,6 @@ define HAL_BATT_CURR_SCALE 40
 define HAL_BATT2_VOLT_PIN 10 
 define HAL_BATT2_CURR_PIN 11 
 define HAL_BATT2_VOLT_SCALE 34 
-define HAL_BATT2_CURR_SCALE 40 
\ No newline at end of file
+define HAL_BATT2_CURR_SCALE 40 
+
+define AP_PARAM_MAX_EMBEDDED_PARAM 512
```

Comment:
I believe this should all work as you expect if you remove this line.

There's an obscure reason you might want this line, but if you don't know what that reason is you might want to remove this line and save the 700-odd bytes of flash on this board.

Reply:
OK, deleting the line AP_PARAM_MAX_EMBEDDED_PARAM 512 will not affect the saving of parameters, right?